### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by aamster**
This PR contains a hodge podge of improvements.

- Rename create_train_test_split.py to create_dataset.py as it also tallies votes
- Modify vote tallying logic to consider all votes for a given ROI, rather than just the first region it appears in
- Allows passing input means/stds, needed for training from scratch
- Adds `is_trial_run` flag, to just do a single train/val split instead of k-fold validation
- Bugfix for `freeze_to_layer == None`. Before it was freezing all layers when it was `None`
- Adds the ability to specify a specific set of validation splits, rather than splitting the train set using a seed

### CODE REVIEW

File: src/deepcell/cli/modules/cloud/train.py [hunk 1]
```diff
@@ -47,13 +56,15 @@ def run(self):
             volume_size=self.args['volume_size'],
             output_dir=self.args['train_params']['save_path'],
             mlflow_server_uri=tracking_params['mlflow_server_uri'],
-            mlflow_experiment_name=tracking_params['mlflow_experiment_name']
+            mlflow_experiment_name=tracking_params['mlflow_experiment_name'],
+            seed=1234
```
[the remark this answers is not in the document]
Comment (aamster) [reply src/deepcell/cli/modules/cloud/train.py:60]: I'm hardcoding it because I made the mistake of not passing a seed and it caused problems. If the seed is optional, it could happen again.
